README.md

```diff
@@ -25,22 +25,27 @@ wasm-bindgen = { version = "0.2" }
 ```rust
 use serde::{Deserialize, Serialize};
 use tsify::Tsify;
+use tsify::Ts;
 use wasm_bindgen::prelude::*;
+use wasm_bindgen::JsError;
```

Comment:
Side note, I am actually the creator of `JsError` and the non-memory-leaking Result return-throw in wasm-bindgen :)